bgpd/bgp_vty.c

```diff
@@ -1774,6 +1782,29 @@ DEFPY (bgp_suppress_fib_pending,
 	return CMD_SUCCESS;
 }
 
+DEFPY (bgp_global_asnotation,
+       bgp_global_asnotation_cmd,
+       "[no] bgp as-notation [<dot|dot+|plain>$format]",
+       NO_STR
+       BGP_STR
+       "AS number notation format\n"
+       "AS dot format uses A.B for AS number values > 65535\n"
+       "AS dot+ format uses A.B for all AS number values\n"
+       "AS plain format uses an unsigned long value\n")
+{
+	bool changed;
+
+	if (no)
+		changed = bm_asnotation_set_format("plain");
+	else
+		changed = bm_asnotation_set_format(format);
```

Comment:
Since the notation is optional, it means I can enter just `bgp as-notation`? What would be the result here? We should check for this case, IMO.

Reply:
ok